python/tk_multi_publish2/util.py

```diff
@@ -619,7 +619,7 @@ def get_conflicting_publishes(context, path, publish_name, filters=None):
 
     # now build up the filters to match against
     publish_filters = [filters] if filters else []
-    for field in ["code", "entity", "name", "project", "task"]:
```

Comment:
When would we have conflicting publishes that came from a different task?

Reply:
There's nothing restricting different tasks from having the same name, which would lead to them having the same publish path.
I do not like the workflow, but there is nothing in shotgun preventing this, so we should cover the possibility.

Reply:
That's disappointing.